thoth/user_api/api_v1.py

```diff
@@ -291,6 +291,21 @@ def list_thoth_container_images(
         entries.append(item)
 
     prev_page, next_page = _compute_prev_next_page(page, per_page)
+
+    if len(entries) == 0:
+        return (
+            {"error": "No Image found", "parameters": parameters},
+            404,
+            {
+                "page": page,
+                "per_page": per_page,
+                "page_count": page_count,
+                "entries_count": entries_count,
+                "next": next_page,
+                "prev": prev_page,
```

Comment:
I don't think this dictionary is necessary here.

Reply:
ack